assets/blocks/BouncyBlock.block

```diff
@@ -0,0 +1,19 @@
+{
+    "displayName": "Bouncy block",
+    "attachmentAllowed" : true,
+    "shadowCasting" : false,
+    "rotation" : "horizontal",
+    "categories" : ["BouncyBlock"],
+    "hardness": 15,
+    "tiles": {
+      "top": "funnyBlocks:BouncyBlock",
```

Comment:
You can actually take out `top` and `bottom` - since the block name matches the texture that'll be the default anyway :-)